.github/actions/setup-rust-runtime/action.yaml

```diff
@@ -34,5 +34,5 @@ runs:
         echo "RUSTC_WRAPPER=sccache" >> $GITHUB_ENV  
         echo "SCCACHE_GHA_ENABLED=true" >> $GITHUB_ENV 
         echo "RUST_BACKTRACE=1" >> $GITHUB_ENV
-        echo "RUSTFLAGS=-C debuginfo=line-tables-only -C incremental=false" >> $GITHUB_ENV
+        echo "RUSTFLAGS=-C debuginfo=line-tables-only -C incremental=false -Wunused-crate-dependencies" >> $GITHUB_ENV
```

Comment:
if this is global lint, it could be defined in cargo.toml in the lint section.
However, this is awesome for production code, but may be hard to do for tests/benchmark code.
At least in arrow-rs, i tried to enable this globally but failed.
see https://github.com/apache/arrow-rs/pull/6804#issuecomment-2511150149

Reply:
going through the same. I'm keen to disable the lint for benches lets see where it can bring me